tools/sof-kernel-log-check.sh

```diff
@@ -189,8 +188,8 @@ ignore_str="$ignore_str"'|\[drm:drm_dp_send_link_address \[drm_kms_helper\]\] \*
 
 # CHT devices with USB hub, issues reported by sof-test
 # BugLink: https://github.com/thesofproject/sof-test/issues/431
-ignore_str="$ignore_str"'|hub 2-.: .'
-ignore_str="$ignore_str"'|usb 2-.: .'
+ignore_str="$ignore_str"'|hub [23]-.+'
+ignore_str="$ignore_str"'|usb [23]-.+: Failed to suspend device, error'
```

Comment:
I think the first line ignores everything that the second line does, so the second line is pointless.

The first line has two issues: 1. it ignores all USB errors on devices `2-9: ...`, it ignores zero error on devices like `2-10: ...`

> As we want to ignore all hub error but not all usb errors, ...

I don't think there is any concept of "hub error" vs "other errors", all errors are prefixed the same.

Can you please ignore only the errors reported in #431 ?